modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,188 @@
+
+[,bash]
+----
+curl -H 'Content-Type: application/json' \
+-H 'Authorization: Bearer <token>' \
+-d '{
+  "name": "<resource-group-name>"
+}' -X POST https://api.redpanda.com/v1beta2/resource-groups
+----
+
+A resource group ID is returned. Pass this ID later when you call the Create Cluster endpoint.
+
+=== Create a network
+
+Create a network by making a request to xref:api:ROOT:cloud-api.adoc#post-/v1beta2/networks[`POST /v1beta2/networks`].
+
+Choose a CIDR range that does not overlap with your existing VPCs or your Redpanda network.
+
+[,bash]
+----
+curl -d \
+'{
+  "cidr_block": "10.0.0.0/20",
+  "cloud_provider": "CLOUD_PROVIDER_GCP",
+  "cluster_type": "TYPE_DEDICATED",
+  "name": "<network-name>",
+  "resource_group_id": "<resource-group-id>",
+  "region": "us-west1"
+}' -X POST https://api.redpanda.com/v1beta2/networks 
+----
+
+This endpoint returns a <<long_running_operations,long-running operation>>. 
+
+=== Create a new cluster
+
+After the network is created, make a request to the xref:api:ROOT:cloud-api.adoc#post-/v1beta2/clusters[`POST /v1beta2/clusters`] with the resource group ID and network ID in the request body. 
+
+[,bash]
+----
+curl -d \
+'{
+  "cloud_provider": "CLOUD_PROVIDER_GCP",
+  "connection_type": "CONNECTION_TYPE_PUBLIC",
+  "name": "my-new-cluster",
+  "resource_group_id": "<resource-group-id>",
+  "network_id": "<network-id>",
+  "region": "us-west1",
+  "throughput_tier": "tier-1-gcp-um4g",
+  "type": "TYPE_DEDICATED",
+  "zones": [
+    "us-west1-a",
+    "us-west1-b",
+    "us-west1-c"
+    ]
+  }
+}' -X POST https://api.redpanda.com/v1beta2/clusters
+----
+
+The Create Cluster endpoint returns a <<long_running_operations,long-running operation>>. When the operation is completed, you can retrieve cluster details by calling xref:api:ROOT:cloud-api.adoc#get-/v1beta2/clusters/-id-[`GET /v1beta2/clusters/{id}`], passing the cluster ID as a parameter.
+
+=== Additional steps to create a BYOC cluster
+
+. Ensure that you have installed rpk.
+. After making a Create Cluster request, run the `rpk cloud byoc` command. Pass `metadata.cluster_id` from the Create Cluster response:
++
+[tabs]
+====
+GCP::
++
+--
+```bash
+rpk cloud byoc gcp apply --redpanda-id=<metadata.cluster_id> --project-id=<gcp-project-id>
+```
+--
+AWS::
++
+--
+```bash
+rpk cloud byoc aws apply --redpanda-id=<metadata.cluster_id>
+```
+--
+====
+
+== Delete a cluster
+
+To delete a cluster, make a request to the xref:api:ROOT:cloud-api.adoc#delete-/v1beta2/clusters/-id-[`DELETE /v1beta2/clusters/{id}`] endpoint, passing the cluster ID as a parameter. This is a <<long_running_operations,long-running operation>>.
+
+```bash
+curl -X DELETE https://api.redpanda.com/v1beta2/clusters/<cluster_id>
+```
+
+=== Additional steps to delete a BYOC cluster
+
+. Make a request to xref:api:ROOT:cloud-api.adoc#get-/v1beta2/clusters/-id-[`GET /v1beta2/clusters/{id}`] to check the state of the cluster. Wait until the state is `STATE_DELETING_AGENT`.
```

Comment:
no action required at the moment:
@sago2k8 we should get this into the list for GA: rpk should be able to accept DELETING state already, and wait if necessary